### PR TITLE
UI/UX - Interview Preparation

### PR DESCRIPTION
**Pull request by karthxk07**
### Fixes Issue
#104 

### Description:
Added Resources in the Interview preparation in UI/UX Category.

### Checklist:

- [X] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [X] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by karthxk07** [comment]
@jfmartinz are the resources fine

File: UI-UX Design/README.md [hunk 1]
```diff
@@ -279,6 +279,18 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
       <td><a href="https://youtube.com/playlist?list=PL1wFSWLYg_Ia97VL4pv-Ovz9330HfZX_k&si=1cHzpetgv_-0uszp">Webdesign Interviews/Podcast</a></td>
       <td>"Welcome to the 'Web Design Business Podcast Interviews' channel. On this channel, you'll find content covering the business aspects of web design, video interviews from podcast, and live stream Q&A sessions, with new videos typically posted 1-2 times a week.</td>
       </tr>
+      <tr>
+      <td><a href="https://github.com/h5bp/Front-end-Developer-Interview-Questions">Front-end Developer Interview Questions</a></td>
+      <td>A GitHub repository with a collection of front-end developer interview questions that can be useful for web design interviews.</td>
+      </tr>
+       <tr>
+      <td><a href="https://interviewing.io/">Interviewing.io</a></td>
+      <td>A platform that offers anonymous technical interview practice with experienced engineers from top tech companies.</td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:288]: These resources are not related for the category UI & UX design.  You can check this [resource](https://www.youtube.com/playlist?list=PL1wFSWLYg_Ia97VL4pv-Ovz9330HfZX_k) currently in the interview section

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: UI-UX Design/README.md [hunk 2]
```diff
@@ -279,6 +279,10 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
       <td><a href="https://youtube.com/playlist?list=PL1wFSWLYg_Ia97VL4pv-Ovz9330HfZX_k&si=1cHzpetgv_-0uszp">Webdesign Interviews/Podcast</a></td>
       <td>"Welcome to the 'Web Design Business Podcast Interviews' channel. On this channel, you'll find content covering the business aspects of web design, video interviews from podcast, and live stream Q&A sessions, with new videos typically posted 1-2 times a week.</td>
       </tr>
+      <tr>
+      <td><a href="https://github.com/h5bp/Front-end-Developer-Interview-Questions">Front-end Developer Interview Questions</a></td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:283]: I am sorry bro. I didnt see that this one is allso not accepted as a resource. Make sure it is related to UI UX design

Comment (jfmartinz) [reply UI-UX Design/README.md:283]: Add something related to UUI UX design and I will merged it Once you completed it thanks

**Comment by jfmartinz** [comment]
Something like [this](https://www.youtube.com/watch?v=2OzTi0iUMlE) if you have trouble finding one

**Comment by karthxk07** [comment]
Could i add this only , if it is ok?

**Comment by karthxk07** [comment]
@jfmartinz I have made the changes accordingly, could you please check and mege. :relaxed:

**Review by jfmartinz — APPROVED** [review]
Looks good! Thanks @karthxk07

**Pull request merged**